apps/backend/src/auth/auth.service.ts

```diff
@@ -82,16 +82,22 @@ export class AuthService {
 
   verifyUser(email: string, verificationCode: string): Promise<unknown> {
     return new Promise((resolve, reject) => {
-      return new CognitoUser({
```

Comment:
❓Do the changes in this method affect its behavior in any way or are these changes just to make the method easier to read?